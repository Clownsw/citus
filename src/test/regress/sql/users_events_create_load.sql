CREATE TABLE users_table (user_id int, time timestamp, value_1 int, value_2 int, value_3 float, value_4 bigint);
SELECT create_distributed_table('users_table', 'user_id');

CREATE TABLE events_table (user_id int, time timestamp, event_type int, value_2 int, value_3 float, value_4 bigint);
SELECT create_distributed_table('events_table', 'user_id');

COPY users_table FROM STDIN WITH CSV;
6,2017-11-23 00:09:44.19812,5,2,0,
4,2017-11-23 00:28:45.060248,4,4,3,
5,2017-11-22 22:31:47.62577,3,1,4,
2,2017-11-23 01:14:27.658529,4,4,4,
1,2017-11-23 17:30:34.635085,3,4,4,
4,2017-11-23 02:14:35.047974,4,4,1,
3,2017-11-23 11:18:53.114408,2,2,0,
3,2017-11-22 18:43:51.450263,1,1,4,
5,2017-11-23 16:48:32.08896,5,2,1,
5,2017-11-23 09:05:08.53142,2,2,2,
2,2017-11-23 06:01:08.148777,2,4,2,
2,2017-11-23 00:19:14.138058,3,4,0,
5,2017-11-23 02:50:32.678074,4,2,4,
5,2017-11-23 14:04:49.464065,4,0,0,
4,2017-11-23 15:32:02.360969,5,3,1,
2,2017-11-23 01:04:26.198826,4,3,4,
2,2017-11-23 12:23:16.53276,1,3,3,
4,2017-11-23 12:59:23.28005,4,1,3,
5,2017-11-22 20:43:18.667473,0,3,2,
5,2017-11-23 00:54:44.192608,1,3,2,
3,2017-11-23 00:15:45.610845,1,1,4,
3,2017-11-23 03:23:24.702501,1,2,5,
5,2017-11-23 09:17:14.659809,3,1,1,
2,2017-11-23 11:48:24.943542,0,5,5,
4,2017-11-23 11:35:24.161296,0,4,1,
4,2017-11-23 07:18:43.255734,3,4,1,
5,2017-11-22 21:02:07.575129,2,0,2,
2,2017-11-23 08:22:22.169158,4,2,5,
5,2017-11-23 00:46:13.498577,3,2,2,
5,2017-11-23 07:47:09.542999,1,4,3,
2,2017-11-23 06:50:30.797805,1,1,1,
3,2017-11-23 11:31:17.403189,4,5,3,
6,2017-11-23 14:43:18.024104,3,2,5,
1,2017-11-23 11:11:24.40789,3,4,0,
4,2017-11-23 06:50:08.101207,2,1,5,
5,2017-11-23 16:28:38.455322,2,5,4,
6,2017-11-22 20:15:53.317797,1,1,1,
4,2017-11-23 10:58:31.388305,0,2,3,
4,2017-11-23 01:55:21.824618,3,1,4,
4,2017-11-22 23:48:11.949567,2,0,0,
4,2017-11-23 07:28:42.537255,3,5,3,
3,2017-11-23 11:41:21.157066,3,5,3,
3,2017-11-23 17:10:35.959913,4,3,1,
1,2017-11-22 22:51:43.132261,4,0,3,
4,2017-11-23 08:38:45.877401,4,1,4,
4,2017-11-23 09:08:27.034145,0,0,3,
1,2017-11-23 09:26:42.145043,1,3,3,
5,2017-11-22 22:10:24.315371,1,2,1,
6,2017-11-23 13:51:16.92838,0,4,2,
2,2017-11-23 13:52:54.83829,3,1,4,
2,2017-11-23 11:47:26.900284,2,0,4,
2,2017-11-23 09:59:08.975079,2,2,4,
2,2017-11-23 03:27:50.327051,2,2,0,
6,2017-11-23 00:07:11.068353,1,1,4,
3,2017-11-22 23:24:32.080584,3,2,5,
1,2017-11-23 11:44:57.515981,4,3,4,
4,2017-11-23 14:57:36.278713,3,1,5,
3,2017-11-23 05:01:44.885505,3,5,4,
3,2017-11-22 20:43:31.008625,1,3,2,
2,2017-11-23 08:49:47.029236,4,2,4,
3,2017-11-23 12:56:49.29191,0,5,1,
1,2017-11-23 17:23:03.441394,5,4,3,
5,2017-11-23 06:45:00.678091,3,2,3,
3,2017-11-23 17:18:51.048758,1,5,5,
6,2017-11-23 01:14:55.769581,0,0,5,
4,2017-11-22 23:59:46.493416,3,1,3,
3,2017-11-23 09:57:41.540228,2,2,3,
4,2017-11-23 06:39:06.287818,3,3,2,
2,2017-11-23 06:23:53.572592,4,4,5,
4,2017-11-22 21:33:03.616802,5,4,1,
5,2017-11-23 09:17:47.706703,2,5,3,
3,2017-11-23 04:01:08.04806,5,5,3,
4,2017-11-23 03:34:40.419294,1,0,4,
6,2017-11-23 11:08:04.244582,2,3,2,
5,2017-11-23 11:33:17.739772,4,3,2,
5,2017-11-22 23:10:42.777699,3,4,5,
5,2017-11-23 10:46:56.036776,3,1,3,
2,2017-11-23 06:56:38.46819,0,1,3,
4,2017-11-23 11:45:39.744961,4,5,4,
5,2017-11-23 14:29:02.557934,2,1,2,
5,2017-11-23 15:55:08.493462,0,3,3,
4,2017-11-23 07:09:37.382372,4,1,2,
4,2017-11-23 13:56:39.967304,0,1,5,
4,2017-11-23 05:42:12.89386,2,3,3,
6,2017-11-23 01:13:50.526322,2,4,1,
2,2017-11-23 09:54:28.13665,0,3,4,
3,2017-11-23 03:52:32.008895,4,2,0,
2,2017-11-22 18:19:49.944985,3,5,1,
6,2017-11-23 10:22:11.02918,5,0,5,
5,2017-11-23 10:15:31.764558,2,2,2,
1,2017-11-23 03:32:50.803031,3,2,1,
5,2017-11-23 06:35:05.166535,5,5,1,
5,2017-11-23 02:09:42.27857,3,2,4,
4,2017-11-23 11:22:36.283022,4,1,4,
6,2017-11-22 23:01:24.82289,2,4,1,
3,2017-11-23 12:43:49.261057,4,3,1,
2,2017-11-23 11:41:04.042936,3,4,1,
5,2017-11-23 10:22:39.468816,3,4,3,
5,2017-11-23 06:36:11.898621,5,1,5,
5,2017-11-23 14:19:45.144019,3,3,0,
3,2017-11-23 06:20:05.854857,1,4,2,
\.

COPY events_table FROM STDIN WITH CSV;
2,2017-11-23 12:32:57.851419,3,3,4,
2,2017-11-23 03:35:04.321504,1,0,2,
5,2017-11-22 21:24:22.849224,5,4,1,
6,2017-11-23 14:00:13.20013,3,3,3,
4,2017-11-23 16:20:33.264457,0,0,3,
4,2017-11-23 04:01:12.29256,4,4,3,
2,2017-11-23 17:26:14.563216,1,5,3,
2,2017-11-22 20:16:16.614779,0,1,1,
3,2017-11-22 21:12:24.542921,1,2,5,
3,2017-11-23 02:03:45.966467,4,1,0,
2,2017-11-23 13:27:37.441959,5,2,4,
4,2017-11-23 18:10:21.338399,1,2,4,
5,2017-11-23 09:20:13.303927,3,0,3,
3,2017-11-22 22:05:38.409323,3,2,2,
5,2017-11-23 14:28:51.833214,3,0,1,
3,2017-11-23 07:11:12.987954,3,1,3,
4,2017-11-23 03:20:23.803407,2,3,3,
3,2017-11-22 20:23:46.906523,3,3,2,
3,2017-11-23 03:02:35.021717,1,0,2,
4,2017-11-23 03:29:32.271967,3,2,4,
6,2017-11-22 20:36:09.106561,3,2,1,
5,2017-11-23 16:11:02.929469,4,2,0,
3,2017-11-23 18:08:26.550729,2,4,3,
2,2017-11-22 22:23:25.40611,3,4,4,
1,2017-11-22 23:22:09.957743,1,1,1,
4,2017-11-23 14:19:25.765876,3,1,1,
3,2017-11-22 21:26:21.185134,1,5,3,
3,2017-11-23 14:00:12.884702,3,1,1,
4,2017-11-23 07:19:09.381749,1,0,2,
5,2017-11-22 22:45:17.377228,3,2,0,
3,2017-11-23 04:45:23.548548,1,2,3,
4,2017-11-22 19:00:10.396739,2,1,1,
4,2017-11-23 08:36:53.871919,2,4,3,
6,2017-11-23 00:45:41.784391,2,3,2,
4,2017-11-22 21:05:25.194441,5,4,1,
3,2017-11-23 09:38:45.338008,2,5,0,
3,2017-11-23 01:17:49.040685,2,3,4,
1,2017-11-22 21:06:57.457147,4,3,2,
6,2017-11-22 21:17:09.549341,5,2,5,
4,2017-11-23 00:40:52.517603,1,5,4,
4,2017-11-23 08:14:18.231273,4,3,2,
1,2017-11-22 19:07:03.846437,1,2,5,
2,2017-11-23 14:02:47.738901,1,3,2,
1,2017-11-23 09:33:16.992454,3,4,1,
5,2017-11-23 14:40:40.467511,1,4,1,
2,2017-11-23 01:08:57.24208,2,3,1,
2,2017-11-22 22:56:47.673504,4,1,4,
2,2017-11-22 22:50:33.855696,4,2,3,
5,2017-11-23 06:56:03.019565,2,1,1,
6,2017-11-23 00:01:48.155345,4,2,1,
6,2017-11-23 02:06:53.132461,5,1,1,
2,2017-11-23 05:42:25.859537,2,3,5,
4,2017-11-23 12:58:37.035024,1,2,4,
1,2017-11-23 11:09:38.074595,0,5,1,
5,2017-11-23 14:23:09.889786,3,1,0,
5,2017-11-22 20:45:35.99031,1,2,3,
3,2017-11-23 03:53:44.925122,4,2,3,
1,2017-11-23 05:19:55.213875,0,2,0,
1,2017-11-22 19:03:01.772353,4,1,2,
5,2017-11-23 02:27:02.996588,1,4,1,
2,2017-11-23 07:09:08.06934,1,5,4,
2,2017-11-23 10:14:51.816339,4,2,1,
3,2017-11-23 13:30:04.643655,3,2,1,
2,2017-11-23 02:06:28.579787,3,1,4,
4,2017-11-23 02:59:28.548578,4,2,3,
2,2017-11-22 22:06:12.107108,0,2,5,
2,2017-11-23 10:46:34.177173,3,2,3,
2,2017-11-23 08:26:23.045769,1,1,0,
3,2017-11-23 06:44:50.887182,4,0,4,
6,2017-11-22 22:44:48.458334,1,3,2,
6,2017-11-22 23:15:15.875499,4,3,3,
3,2017-11-23 16:44:41.903713,4,2,2,
5,2017-11-23 13:02:31.055159,0,2,1,
4,2017-11-23 07:32:45.521278,4,5,3,
5,2017-11-23 03:23:09.888244,1,3,2,
1,2017-11-23 09:23:30.994345,3,1,1,
1,2017-11-22 20:56:21.122638,2,4,4,
1,2017-11-23 02:59:23.620864,4,5,4,
2,2017-11-23 10:59:25.787633,3,1,0,
5,2017-11-23 13:26:45.571108,3,3,4,
2,2017-11-23 06:23:43.305403,1,2,1,
4,2017-11-23 04:32:21.986825,4,1,3,
1,2017-11-22 21:47:04.188168,4,2,0,
4,2017-11-23 06:34:15.714083,2,2,2,
2,2017-11-23 04:05:16.217731,1,4,3,
2,2017-11-23 12:17:48.482544,2,3,2,
2,2017-11-23 15:58:49.273421,5,1,2,
2,2017-11-23 12:06:39.464165,4,5,4,
3,2017-11-23 16:31:56.219594,5,1,2,
1,2017-11-23 10:23:27.617726,4,2,5,
3,2017-11-23 10:24:02.530471,2,2,3,
6,2017-11-23 07:27:32.822068,2,2,1,
5,2017-11-23 04:16:29.402814,1,3,3,
3,2017-11-23 13:19:47.138294,2,1,4,
2,2017-11-23 08:07:58.727599,3,1,3,
1,2017-11-23 00:42:37.237615,2,4,3,
3,2017-11-22 22:54:59.938341,2,4,3,
6,2017-11-23 11:16:13.106691,1,1,0,
3,2017-11-22 18:36:16.372893,2,3,4,
1,2017-11-22 18:49:42.327403,3,2,1,
1,2017-11-23 21:54:46.924477,6,4,5,
\.
