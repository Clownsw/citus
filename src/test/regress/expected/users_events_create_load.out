CREATE TABLE users_table (user_id int, time timestamp, value_1 int, value_2 int, value_3 float, value_4 bigint);
SELECT create_distributed_table('users_table', 'user_id');
 create_distributed_table
---------------------------------------------------------------------

(1 row)

CREATE TABLE events_table (user_id int, time timestamp, event_type int, value_2 int, value_3 float, value_4 bigint);
SELECT create_distributed_table('events_table', 'user_id');
 create_distributed_table
---------------------------------------------------------------------

(1 row)

COPY users_table FROM STDIN WITH CSV;
COPY events_table FROM STDIN WITH CSV;
